CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.6.11 - 2019-10-21
```

Comment:
nit: could we update the `Versioning and Deploy` section of the README to explain this CHANGELOG.md file also needs to be updated when a new version is created?

Reply:
good idea, updated.